### PR TITLE
Backend deployment

### PR DESCRIPTION
**Pull request by bonvee-99**

### CODE REVIEW

File: backend/src/status.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    } else {
+        return {
+            statusCode: status || 500,
+            body: "Essentially is active but Summary API is down :("
+        }
+    }
+  } catch {
+    return {
+      statusCode: 500,
+      body: "Essentially is active but Summary API is down :("
```
Comment (Michaeltaoma) [inline backend/src/status.ts:31]: Correct me if I am wrong, but it looks like the message here should be "Essentially is down..."?

Comment (bonvee-99) [reply backend/src/status.ts:31]: agreed. It was like this for the express api but i think it should be that

Comment (bonvee-99) [reply backend/src/status.ts]: pushed new commit with change

File: backend/package.json [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
-    "@types/isomorphic-fetch": "^0.0.36",
-    "cors": "^2.8.5",
-    "express": "^4.18.2",
-    "jsdom": "^20.0.2"
+    "aws-lambda": "^1.0.7",
+    "aws-sdk": "^2.1323.0"
   },
   "scripts": {
+    "build": "tsc",
     "start": "tsc && node dist/app.js",
```
Comment (Michaeltaoma) [inline backend/package.json:9]: Is the `start` command still in use? If not, can we remove this?

**Review by Michaeltaoma — COMMENTED** [review]
Nicely done! Is this already being deployed? If so, could you include the domain name and how we can interact with it in some documentation? Also, I was wondering if we could test the lambda function locally.

File: backend/package.json (hunk 2 above)
Comment (bonvee-99) [reply backend/package.json:9]: yes will remove

**Review by Michaeltaoma — DISMISSED** [review]

**Comment by Michaeltaoma** [comment]
The lambda function is working already! However, since our front-end code still consumes the old express backend, merging this change will disable our front end. I was wondering if we have a plan to migrate our front end to consume the lambda endpoint as well.

**Review by armintalaie — APPROVED** [review]

**Pull request merged**